css/style.css

```diff
@@ -0,0 +1,89 @@
+body {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.container {
+    padding: 2rem;
+    margin: 1rem 3rem 3rem 3rem;
+}
+
+.navbar {
+    padding: 0 3rem;
+    border-bottom: 2px solid var(--color-primary);
+}
+
+.input-icon {
+    position: absolute;
+    top: -5%;
+    padding: 0.625rem;
+    color: var(--color-primary);
+}
+
+.navbar>.form-control>.input {
+    padding: 10px;
+}
+
+.card-vertical-md>.card-img {
+    height: 18.75rem;
+}
+
+.card-img-1 {
+    background-image: url('../img/got.jpg');
+    background-size: cover;
+    background-position: center top;
+    height: 18.75rem;
+}
+
+.display-center {
+    display: flex;
+    justify-content: center;
+}
+
+.card-img-2 {
+    background-image: url('../img/BB.png');
+    background-size: cover;
+    background-position: center;
+    height: 18.75rem;
+}
+
+.rule-box {
+    margin: 10px auto;
+    padding: 1.5rem;
+    width: 37.5rem;
+    border: 2px solid var(--color-primary);
+}
+
+.rule-list {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 9.5rem;
+}
+
+.card-body {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+}
+
+.flex {
+    display: flex;
+    flex-direction: row;
```

Comment:
flex direction is row by default. You can remove this unless you are using it for overriding.